contrib/test_mergeBenchmarkSets/test_integration/__init__.py

```diff
@@ -0,0 +1,157 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import filecmp
+import os
+import subprocess
+import sys
+import tempfile
+import unittest
+
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
+
+here = os.path.relpath(os.path.dirname(__file__))
+base_dir = os.path.join(here, "..", "..", "..")
+merge_benchmark_sets = [
+    sys.executable,
+    os.path.join(base_dir, "contrib", "mergeBenchmarkSets.py"),
+]
```

Comment:
Couldn't we just run the main function from `mergeBenchmarkSets` instead of spawning subprocesses?
For handling stdout one could e.g. use contextlib. redirect_stdout.